components/net/fetch/methods.rs

```diff
@@ -1148,8 +1148,14 @@ fn http_network_fetch<UI: 'static + UIProvider>(request: Rc<Request>,
     // TODO this step isn't possible yet
     // Step 13
 
+    // Step 14.
+    if credentials_flag {
+        // TODO: should block cookies?
```

Comment:
I don't think this is a meaningful TODO given https://tools.ietf.org/html/rfc6265#section-7.